boards/arm/nucleo_f302r8/nucleo_f302r8.dts

```diff
@@ -8,6 +8,7 @@
 #include <st/f3/stm32f302X8.dtsi>
 #include <st/f3/stm32f302r(6-8)tx-pinctrl.dtsi>
 #include "arduino_r3_connector.dtsi"
+#include "st_morpho_header.dtsi"
```

Comment:
Just a non-blocking comment, this kind of file is generally called "_connector.dtsi". Unless you have a strong aversion for this wording, it would be nice to adhere to this "convention" to avoid seeing all sorts of variants poping up in future.

Reply:
Agreed.

Reply:
renamed to `_connector.dtsi`